README.md

```diff
@@ -57,13 +90,14 @@ ck0i0krq457c7bgte4l0:
 Set the vault token for use in the client:
 
 ```bash
-export VAULT_TOKEN=hvs.Z3CuzSQno3XMuUgUcm1CmjQK
+export VAULT_TOKEN=hvs.WjJyFMP0jvNwBgPgelOcO4BM
+
 ```
 
 Read the `vault-ca-certificate` secret content:
 
 ```bash
-user@ubuntu:~$ juju show-secret ck0i0h3q457c7bgte4kg --reveal
+user@ubuntu:~$ juju show-secret ckp5ud41tqvs77qu9ga0 --reveal
 ck0i0h3q457c7bgte4kg:
   revision: 1
   owner: vault-k8s
```

Comment:
I tried running through this README.md and I'm getting the following error when running any vault command (ex. `vault status`):

```
Error checking seal status: Get "https://vault.poulah.com/vault-vault-k8s/v1/sys/seal-status": x509: certificate signed by unknown authority
```

Reply:
README.md is updated.  ca_certificate is not correct above. We need to pass ca_certificate which is taken from self-signed-certificates operator.
```
venv) gatici@gaticipc:~/charmed5g/vault-k8s-operator$ juju show-secret cks0s1c24l7c77v23p80 --reveal
cks0s1c24l7c77v23p80:
  revision: 1
  expires: 2024-10-23T18:33:41Z
  owner: self-signed-certificates
  label: ca-certificates
  created: 2023-10-24T18:33:42Z
  updated: 2023-10-24T18:33:42Z
  content:
    ca-certificate: |
      -----BEGIN CERTIFICATE-----
      MIIDZzCCAk+gAwIBAgIUOk97hMSxCC8K2012ASoqXSVWd8IwDQYJKoZIhvcNAQEL
      BQAwOTELMAkGA1UEBhMCVVMxKjAoBgNVBAMMIXNlbGYtc2lnbmVkLWNlcnRpZmlj
      YXRlcy1vcGVyYXRvcjAeFw0yMzEwMjQxODMzNDFaFw0yNDEwMjMxODMzNDFaMDkx
      CzAJBgNVBAYTAlVTMSowKAYDVQQDDCFzZWxmLXNpZ25lZC1jZXJ0aWZpY2F0ZXMt
      b3BlcmF0b3IwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDc+mVu8Wmg
      NNV8lhTXt+0l8ttttIqZJvTxe3iwR29ywVbivpyGqzssS/jw3126mhj9Fc9dE2hl
      QnehDUge+FmuwCzPXroFzRf40ierji8WDTfCZTFFLVBPQt/I1yNhriI09gpfrCxP
      QXZpx4sotF42Lt6dXI2DTj0TdDIvy7AlD+Xo7iuAqg0Cv+5GSBNiKQNKi/GHrKrl
      2M4FAGaqjB+UxINlfkKW2VSJcf4cup6HR4PW1cPnTZmxzvnCfFHPRY7UdsF2OsdH
      UdQskSXt/j8gjd65cqIF0tgbMu71p9RD7889No7jkzN/aOubvSp+87mLudb2uEeM
      iYxjHx5BPsaDAgMBAAGjZzBlMB8GA1UdDgQYBBYEFLB4tZ8xOYGfWUrCPvVaNYx+
      u9PdMCEGA1UdIwQaMBiAFgQUsHi1nzE5gZ9ZSsI+9Vo1jH67090wDgYDVR0PAQH/
      BAQDAgKkMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBAFkYT508
      cJ2ZqKTVK+bvb8dPyEV6AJK5JthkUndpremCGrUZvBhtNyUW0vzkBbeoSpD2j1IV
      kNFw57foD88UdvhyTLfjx11fW3jZ1OWyeFCEMxlaPSTHwBp7SSpvl+hwi4R8FG5b
      rsroTfKwYLyk75xyZkT93UGszOOpqgEdp/ca2TBHf0QyEQO390XZ2U0uWSce/LfY
      TzrV0NvxulKA/CmWToyuRAwbv74qvZuFF+4j6EgmGINNL6DarnAbIfvN559S1LI/
      FuqFiGZXpOMpoQln6JGmWY7KV9cpTF9iH+FYyqP72r232YTsn7b2cc2NhqYQckYH
      ddn0ebSoPVuitB0=
      -----END CERTIFICATE-----
    private-key: |
      -----BEGIN RSA PRIVATE KEY-----
      Proc-Type: 4,ENCRYPTED
      DEK-Info: AES-256-CBC,BC66FCF8D0E4646E79340DC33B57DBE7

      hao+I0rjORgfI79kkT6w9S/LH0A2nnpaXChxJw/IB25xavnrubGiaQGbf9EIlG9n
      9ULVsN/8xyLkmgEbA91/y40sXnV5h/O/TgJG9Fk+iWOegQpGIe6RQo1GoqDPHvVO
      sKIUknQoUaUNC6LDTauVH/eG7OqogKvL/mPIL4ILJHmUGBl/GG9EVbDzDx0q3DCq
      AxXfAx/BE4e/fweQWQMkxEsaOXpLXscdng4uL/7vCzdIJNenkgjLburyheQudRzI
      iKQ12HgJ7L38uGevlzb5UuqfrxnDPb+uPyE0wkew0gHmZbIU9Ebay1cIBv05zAJB
      OgX2BFqoIrNcLc1znDUgAF2v5NTfClCzbzZG3TPU4ACBQzfI3p24xENUoXEnUFjW
      vuLCC7EiS7zprY0fu1qwG8fYNVsdk6AvD7z2tTdgEcxlQaIEgSlQll5+fK70Jra3
      rbcCWJx1/KRmj08fyFlEJ5OExKVUMOSgeqA1wv+/QOSnqyoqtMDhGsmKarZJhJQG
      u9SNcaTtizUKxdGqdIvF6tHWkDUCcWZ6yQdbgWZW3nyK1EVS8zhRUR5cqncTjVs1
      w2aFpfYemNSgoYA47BD3ZjUm8LvBGEzhBI8bmpKYFwQmjZqW+CC157NJjMwa/WiA
      qN9BeYxm2Sxhqtptzk+8grcrTSnr7KFV5SGHI3mzz9VsZJPrIqOTdjUaa3pMoaUS
      yPuQOVDBhHv1zUAH7ss694tVxjsEM3Zc8JpTWU2xbX9O8VawN/8nQ9lAwgflbmgO
      JwhIgsB+PBh5/Gzgc/Pimjp+ZzsRw0IEOYvTU2vqkmD3DZamwnuaJsmMosUFk4lW
      p4fDIXSkgbEHjnlwL/ZsJRkXS8HPMzQiED9X6eSN93Q3pZKT/cDsApAgXlSERkW5
      QrBX1QlB575KSSyGb6UvaDk1F8kO6BdeEIvtyxciDZ4MoCE2gPcONnDSElXI7aqk
      J0+bgzqWOB/KwV6RNbJdiWPro9euzoUZfIxg3B7CpmmNnhfNMYAITw9cEC81h4ec
      XktIPKj3zyDppDkOXM2eFkYVl1NH6I1bXooHXkkY62+I+DaogSnfAo6Yr15+5ZZ0
      yzU0w2J2Dwp8Bs16w36KTmoZ3lLuFcEVfpPcedJvBqR8nzIVqZEUMCSuwPRE7BMm
      tKFXJ0kGjspAnxNEje1IrUv2OSliOF3Y1Z2ldWBd0GwxkPI+cBIix1ExgBzaHiK3
      Bb/A/XXd7y8eYA0c9SSNSaUHE5ZDQL+P6kwn/9p+l8+MBvyz1d5ggUTcXeUCAyLX
      8zwKmdZMLuh/7OhCVJ19OMB1gwKARSk0moeVFzTM+D6O+qvs2T/R4odJUWKY3F9E
      cKUPxetoiUIptsE3fOydwLt3RNJtQ3PXFokiCsqT5DXG5mNVDe+Svu/bQWPC0au5
      MkzDjFR4gJUskFS8HgrQowSWplwkBawgpAwtXifLgQ7WK1A6CR8A0hGr9TBqxKhQ
      w9FE0kK6ltwTTx/jjpPqWz4XfMiN8JhrB8OgdSeLBgAdQxcCKyPvpM1aiSsnkPB3
      a3MNK20tvTA8J74k+b340YeiDSNycTvByiPKtHiPrEqB5X9B7ugYdIItmLqFe+mB
      -----END RSA PRIVATE KEY-----
    private-key-password: 49acc9f8e1e3351f7e064e1a3405b170b884d5427dfaeeb6af67b5236de6dbaa4b957b27ae8431eeefdbe6392b6589045fa223186cb1a4974e5fdca9353d38ba
(venv) gatici@gaticipc:~/charmed5g/vault-k8s-operator$ export VAULT_CAPATH=/home/gatici/charmed5g/vault-k8s-operator/vault_ca.pem
(venv) gatici@gaticipc:~/charmed5g/vault-k8s-operator$ vault status
Key                     Value
---                     -----
Seal Type               shamir
Initialized             true
Sealed                  false
Total Shares            1
Threshold               1
Version                 1.15.0
Storage Type            raft
Cluster Name            vault-cluster-112a90ce
Cluster ID              db6d5dd6-19df-f377-e901-867823b37beb
HA Enabled              true
HA Cluster              https://10.1.146.63:8201
HA Mode                 active
Active Since            2023-10-24T18:44:37.480998713Z
Raft Committed Index    64
Raft Applied Index      64
(venv) gatici@gaticipc:~/charmed5g/vault-k8s-operator$ 
```

Reply:
Same secret id is kept.